krun.py

```diff
@@ -277,9 +307,32 @@ def main():
     config = util.read_config(config_file)
     out_file = util.output_name(config_file)
 
+    mail_recipients = config.get("MAIL_TO", [])
+    if type(mail_recipients) is not list:
+        fatal("MAIL_TO config should be a list")
+
+    max_mails = config.get("MAX_MAILS", 5)
+
+    # You should only have mailing for real experiments.
+    # Prevent accidental sending of mail during benchmark implementation
```

Comment:
I don't get the point of this if clause. If the user specifies an email address, let's just trust them. Most users hate pressing multiple Y/N stuff, which also prevents automation.